src/main/java/seedu/address/logic/commands/contact/CEditCommand.java

```diff
@@ -56,7 +64,7 @@
     private final EditContactDescriptor editContactDescriptor;
 
     /**
-     * @param index of the contact in the filtered contact list to edit
+     * @param index                 of the contact in the filtered contact list to edit
```

Comment:
there are more than 1 space between 'index' and 'of'

Reply:
This is intended in javadoc. The right side should be aligned. See [this](https://www.oracle.com/sg/technical-resources/articles/java/javadoc-tool.html)